nutanix/resource_nutanix_karbon_cluster.go

```diff
@@ -890,9 +899,15 @@ func flattenNodePools(d *schema.ResourceData, conn *karbon.Client, nodePoolKey s
 	flatNodepools := make([]map[string]interface{}, 0)
 	// start workaround for disk_mib bug GA API
 	expandedUserDefinedNodePools := make([]karbon.ClusterNodePool, 0)
-	var err error
+	// var err error
```

Comment:
Can you remove it if not needed?

Reply:
@marinsalinas Can you review my changes? Thanks!